openlane_update_tool/action.yml

```diff
@@ -0,0 +1,40 @@
+name: Update OpenLane Tool
+
+inputs:
+  tool_name:
+    description: Name of the tool to update.
+    required: true
+  pdk_root:
+    description: Path to the PDK folder.
+    default: .
+  build_ol:
+    description: Build docker image after tool update.
+    default: true
+  ol_tag:
+    description: Tag to use when building the docker image.
+    default: current
+
+runs:
+  using: composite
+  steps:
+
+    - name: Update tool hash
+      shell: bash
+      run: python3 .github/scripts/update_tools.py ${{ inputs.tool_name }}
+
+    - name: Build OpenROAD docker image
```

Comment:
this also look more generic that just OpenROAD (as it can build any tool)

Reply:
Yes, I originally wrote with update only OpenROAD in mind and then changed it to be generic. I missed this name, thanks. Fixed on 8358cfd